pyproject.toml

```diff
@@ -4,7 +4,14 @@ src_paths = ["exporters" ,"scripts"]
 known_first_party = ["pelorus"]
 
 [tool.pytest.ini_options]
+pythonpath = [
+    "exporters/comitttime",
+    "exporters/deploytime",
+    "exporters/failure",
+    "exporters/pelorus"
+]
```

Comment:
I thought we didn't need these. Are you still seeing issues without these?

Reply:
you are right, removed from latest rev.